### ISSUE TITLE
Being able to delete the user profile from the Settings dialog

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS#38243]
You can add or open a user profile from the Settings --> User profile menu.
It could be nice to be able to delete a user profile from that same menu instead of having to do it from the OS browser.

**Comment by wburt** [comment]
Good idea!

**Comment by karl-magnus** [comment]
Agree. I was just searching for this functionality. Seems more logical than going to the OS.

**Comment by tjukanovt** [comment]
+1

I think it would be also nice to have the possibility to select a profile on startup.

**Comment by andreasneumann** [comment]
> I think it would be also nice to have the possibility to select a profile on startup.

I think this depends - for people who change the profiles automatically, it would be convenient - but most people I know don't use profiles and for these, it would be annoying.

Personally, I change profiles quite irregularly and not very often. Mainly for testing purposes.

**Comment by drf5n** [comment]
> You can add or open a user profile from the Settings --> User profile menu. It could be nice to be able to delete a user profile from that same menu instead of having to do it from the OS browser.

So the current work-around is that you should to delete profiles outside of QGIS in the OS?

**Comment by DelazJ** [comment]
@drf5n I'm afraid I don't really understand your feedback here. Yes, managing those folders is the only workaround and asking people to manage these well hidden folders isn't a good UX IMHO. Anyway #58763 is meant to fix the feature request.
